packages/client/src/components/Textarea/Textarea.tsx

```diff
@@ -24,39 +31,58 @@ const Textarea: FC<TextareaProps> = ({
   required,
   error,
   errorMessage,
-  pattern,
   placeholder,
 }) => {
+  const [textareaLength, setTextareaLength] = useState(0);
+  const { isOpen, handleOpenModal, handleCloseModal } = useModal();
+
   const options = {
     ...(required && { required: 'Поле не может быть пустым' }),
-    ...(pattern && { pattern: ValidationPattern[pattern] }),
   };
 
   const { ref, ...rest } = register(name, options);
   const textAreaRef = useRef<HTMLTextAreaElement | null>(null);
-  const textarea = textAreaRef.current;
 
   const combinedRefs = (texteareElement: HTMLTextAreaElement) => {
```

Comment:
Вспомнила, что тут опечатку видела 
```suggestion
  const combinedRefs = (textareaElement: HTMLTextAreaElement) => {
```